draft-js-mention-plugin/src/MentionSuggestions/Entry/Entry.tsx

```diff
@@ -1,7 +1,35 @@
-import React, { useEffect, useRef } from 'react';
+import React, { FC, useEffect, useRef, MouseEvent, ReactElement } from 'react';
 import PropTypes from 'prop-types';
+import { MentionData } from 'draft-js-mention-plugin/src';
+import { Theme } from '../../theme';
 
-const Entry = props => {
+export interface EntryComponentProps {
+  className?: string;
+  onMouseDown(e: MouseEvent): void;
+  onMouseUp(e: MouseEvent): void;
+  onMouseEnter(e: MouseEvent): void;
+  role: string;
+  id: string;
+  'aria-selected'?: string | null;
+  theme?: Theme;
+  mention: MentionData;
+  isFocused: boolean;
+  searchValue?: string;
+}
+
+interface EntryProps {
+  mention: MentionData;
+  entryComponent: FC<EntryComponentProps>;
+  onMentionSelect(v: MentionData): void;
```

Comment:
```suggestion
  onMentionSelect(mention: MentionData): void;
```
Should avoid one letter names in types as well.